std/socket.d

```diff
@@ -122,14 +122,10 @@ else
 
 version(unittest)
 {
-    static assert(is(uint32_t == uint));
-    static assert(is(uint16_t == ushort));
```

Comment:
Note, `uint32_t` and `uint16_t` are nowhere else in this file. I have no idea what these were for so I removed them. If they are critical to have, they should go somewhere else.